src/views/Status.vue

```diff
@@ -32,130 +32,21 @@
-                    </template>
-                    <p class="tooltip-form mb-0">
-                      The estimated time is calculated based on a 10-minute block time.
-                    </p>
-                  </v-tooltip>
-                </v-row>
-                <v-row v-if="!btcConfirmationsAreDone" justify="center" class="mt-2 pa-0">
-                  <h5>
-                    Estimated time left: {{leftBtcTime}} hours
-                  </h5>
-                </v-row>
-              </v-col>
-              <v-col cols="auto" class="pa-0 d-flex justify-center">
-                <div class="img-progress-bar">
-                  <v-row>
-                    <v-img class="d-flex justify-center"
-                           src="@/assets/status/rsk-green.png" height="78" contain/>
-                  </v-row>
-                  <v-row class="mt-4">
-                    <h1>{{environmentContext.getRskText()}} Network</h1>
-                  </v-row>
-                </div>
-              </v-col>
-              <v-col class="confirm-percentage pa-0">
-                <v-row>
-                  <v-progress-linear
-                    :value="rskConfirmationsPercentage"
-                    color="#00B43C"
-                    height="17"/>
-                  <v-row   justify="center" class="mt-4 mx-0 pa-0 mb-0 confirmations-message" >
-                    <h6 v-if="!rskConfirmationsAreDone">
-                      Usually takes around 20 minutes
-                    </h6>
-                  </v-row>
-                </v-row>
-                <div class="d-flex justify-end pa-0 ma-0">
-                  <div class="rbtc-icon-green">
-                    <v-row>
-                      <v-img class="d-flex justify-center"
-                             src="@/assets/status/rbtc_green.png" height="78" contain/>
-                    </v-row>
-                    <v-row class="mt-4">
-                      <h1>{{environmentContext.getRbtcTicker()}} delivered</h1>
-                    </v-row>
-                  </div>
-                </div>
-              </v-col>
-            </v-row>
-          </v-col>
-        </v-row>
       </v-container>
-      <v-row justify="center" v-if="isPegOut" class="mt-6">
-      </v-row>
 
       <v-container fluid class="transactions px-0">
         <!--  TODO: create a pegin-tx-summary component-->
-        <tx-summary
+        <tx-pegin
           v-if="!isRejected && showStatus && isPegIn"
-          :statusFee="currentFee"
-          :statusRefundAddress="currentRefundAddress"
-          :txId="txId"
-          :showTxId="true"
-          :initialExpand="true"/>
-        <!--  TODO: create a pegout-tx-summary component-->
-
+          :txId ="txId"
+          :pegInStatus='pegInStatus'
+          />
+         <!--  TODO: create a pegout-tx-summary component-->
+        <tx-pegout
+          v-if="!isRejected && showStatus && isPegOut"
+          :txId ="txId"
+          :pegStatus="pegOutStatus"
+        />
```

Comment:
May be we don't have to handle the specific `isRejected` field. In these cases we should show the specific error or exception message